src/modules/colorPicker/ColorPickerUI/Keyboard/KeyboardMonitor.cs

```diff
@@ -62,23 +63,21 @@ private void ActivationShortcut_PropertyChanged(object sender, System.ComponentM
 
         private void Hook_KeyboardPressed(object sender, GlobalKeyboardHookEventArgs e)
         {
+            _currentlyPressedKeys.Clear();
```

Comment:
why do you clear pressed keys on every key press and then re-adding them? 
Also this would fail if were to set up shortcut that is something like - Ctrl + A + C - you would be able to capture multiple keys (outside of modifiers)

Reply:
We don't allow such shortcuts in the settings right now.

Reply:
The goal here is to react only to Win/Alt/Ctrl/Shit + key, when the modifiers are pressed first, currently you can press C + Shift + Win in that order and still Color Picker opens.

Reply:
then we should change setting of a shortcut in the settings - I can press C+ Ctrl + Shift  (in this order) and it will set  - Ctrl + Shift + A, giving an impression that I can press buttons in that order?

Reply:
also if you do this, then remove this variable completely, there is no need to hold class variable and clean it every time when the scope is only this method, convert it into local one